extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -34,18 +35,38 @@ export class MigrationCutoverDialogModel {
 			this._migration.subscription,
 			this._migration.migrationContext
 		));
+
+		sendSqlMigrationActionEvent(
+			TelemetryViews.MigrationCutoverDialog,
+			TelemetryAction.MigrationStatus,
+			{
+				'sessionId': this._migration.sessionId ?? '',
+				'migrationStatus': this.migrationStatus.properties.migrationStatus
+			},
+			{}
+		);
 		// Logging status to help debugging.
 		console.log(this.migrationStatus);
 	}
 
 	public async startCutover(): Promise<DatabaseMigration | undefined> {
 		try {
 			if (this.migrationStatus) {
-				return await startMigrationCutover(
+				const cutover = await startMigrationCutover(
 					this._migration.azureAccount,
 					this._migration.subscription,
 					this.migrationStatus
 				);
+				sendSqlMigrationActionEvent(
+					TelemetryViews.MigrationCutoverDialog,
+					TelemetryAction.CutoverMigration,
+					{
+						'sessionId': this._migration.sessionId ?? '',
+						'migrationEndTime': new Date().toString()
+					},
```

Comment:
* migratonEndTime might by misleading, would 'cutoverStartTime' be more accurate.
* would it be safer to call the telemetry event before initiating the call in case it throws?